helpers/base.sh

```diff
@@ -196,28 +168,41 @@ check_python_version() {
     fi
 }
 
-# Chooses a suitable compressor depending on which parameters passed (or didn't
-# pass) the user.
+# Execute a suitable compression depending on which parameters passed (or didn't
```

Comment:
There is a grammatical mistake which the sentence inherits from the original one. It should be substituted for
> Executes a suitable compression depending on which parameters the user passed (or didn't pass).